pkgs/development/tools/continuous-integration/fly/default.nix

```diff
@@ -0,0 +1,40 @@
+{ buildGoModule, fetchFromGitHub, lib, writeText }:
+
+buildGoModule rec {
+  pname = "fly";
+  version = "5.3.0";
+
+  src = fetchFromGitHub {
+    owner = "concourse";
+    repo = "concourse";
+    rev = "v${version}";
+    sha256 = "06ns98k47nafhkkj7gkmxp7msn4ssypyss6ll0fm6380vq2cavnj";
+  };
+
+  modSha256 = "11rnlmn5hp9nsgkmd716dsjmkr273035j9gzfhjxjsfpiax60i0a";
+
+  subPackages = [ "fly" ];
+
+  buildFlagsArray = ''
+    -ldflags=
+      -X github.com/concourse/concourse.Version=${version}
+  '';
+
+  # The fly.bash file included with this derivation can be replaced by a
+  # call to `fly completion bash` once the `completion` subcommand has
+  # made it into a release. Similarly, `fly completion zsh` will provide
+  # zsh completions. https://github.com/concourse/concourse/pull/4012
+  postInstall = ''
+    install -D -m 444 ${
+      writeText "fly.bash" (builtins.readFile ./fly.bash)
+    } $out/share/bash-completion/completions/fly
```

Comment:
```suggestion
    install -D -m 444 fly.bash $out/share/bash-completion/completions/fly
```

Reply:
@kalbasit Because `fly.bash` isn't present in the project's source, I had to implement this slightly differently: https://github.com/NixOS/nixpkgs/pull/64714/commits/9e465d2f60ea404765607c5c7f6d8ad0afae592a#diff-bd215dfd96e2fb6f91323ab49fa3563fR28